src/palette/elements/Histogram/Histogram.stories.tsx

```diff
@@ -0,0 +1,120 @@
+import { storiesOf } from "@storybook/react-native"
+import React from "react"
+import { withTheme } from "storybook/decorators"
+import { Histogram, HistogramBarEntity } from "."
+import { Flex } from ".."
+
+storiesOf("Histogram", module)
+  .addDecorator(withTheme)
+  .add("Simple Histogram", () => (
+    <Flex mx={2} my={2}>
+      <Histogram selectedRange={[0, 50000]} bars={BAR_DUMMY_DATA} />
+    </Flex>
+  ))
+
+const BAR_DUMMY_DATA: HistogramBarEntity[] = [
+  {
+    value: 0,
+    count: 1919,
+  },
+  {
+    value: 2000,
+    count: 845,
+  },
```

Comment:
this is such a tiny thing, and feel free to skip it, but could we format these like:
```suggestion
  { value: 0, count: 1919 },
  { value: 2000, count: 845 },
```